src/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -29,7 +29,7 @@ public sealed partial class Pipe
     }
     public partial class PipeOptions
     {
-        public PipeOptions(System.Buffers.MemoryPool<byte> pool = null, System.IO.Pipelines.PipeScheduler readerScheduler = null, System.IO.Pipelines.PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)0, long resumeWriterThreshold = (long)0, int minimumSegmentSize = 2048, bool useSynchronizationContext = true) { }
+        public PipeOptions(System.Buffers.MemoryPool<byte> pool = null, System.IO.Pipelines.PipeScheduler readerScheduler = null, System.IO.Pipelines.PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)32768, long resumeWriterThreshold = (long)16384, int minimumSegmentSize = 2048, bool useSynchronizationContext = true) { }
```

Comment:
I see the ref was outdated.

Reply:
Yep. I re-ran:
> msbuild /t:GenerateReferenceSource /p:TargetGroup=netstandard /p:ConfigurationGroup=Release

(from src directory of the project)